modules/ROOT/pages/box/box-connector-examples.adoc

```diff
@@ -250,7 +253,295 @@ output application/json
 </mule>
 ----
 
+== Example: Upload a File
+
+This example demonstrates how to upload a file to Box.
```

Comment:
This text adds no value to the title. Is there something more to say, maybe about when or why a user would upload a file to Box?